docs/compile.md

```diff
@@ -3,16 +3,22 @@ Compiler (Solidity)
 
 Clicking the Solidity icon in the icon panel brings you to the Solidty Compiler.
 
-Compiling is triggered when you click the compile button ( **A. in image below**). If you want the file to be compiled each time the file is saved or when another file is selected - check the auto compile checkbox ( **B. in image below**).  
+Compiling is triggered when you click the compile button ( **D. in image below**). If you want the file to be compiled each time the file is saved or when another file is selected - check the auto compile checkbox ( **E. in image below**).
+
+Since the Solidity version `0.5.7`, it is possible to compile `Yul` files. Please read the (([solidity documentation about Yul]https://solidity.readthedocs.io/en/latest/yul.html)) which contain some code examples.
+You can use the language dropdown ( **B. in image below**) to switch the language. It is only activated for a version higher or equal than `0.5.7`.
```

Comment:
"only activated for a version higher or equal than `0.5.7`" this should be in bold I think.

Reply:
"only activated for a version higher or equal than 0.5.7" - should be 
"This dropdown list is only available for versions greater than or equal to 0.5.7"